api/bindings/v1alpha1/bindingconfiguration_types.go

```diff
@@ -104,7 +104,7 @@ type BindingConfigurationStatus struct {
 // BindingEndpoint is a reference to an Endpoint object in the ngrok API that is attached to the kubernetes operator binding
 type BindingEndpoint struct {
 	// Ref is the ngrok API reference to the Endpoint object (id, uri)
-	ngrok.Ref `json:",inline"`
+	v6.Ref `json:",inline"`
```

Comment:
I think this is okay, but generally we've strayed from embedding ngrok-api-go types since it uses snake_case for json serialization([Example](https://github.com/ngrok/ngrok-api-go/blob/2ebad4353c11ffe54afa2a310fa6d6853c6e95dc/datatypes.go#L68)), where as the kubernetes convention is to use camelCase.

Reply:
Talked synchronously about this and we're okay with leaving it for now since both of the fields in here are the same either snake_case or camelCase (`id` and `uri`). Only other thing is this probably belongs under `BindingEndpointStatus`.